### DISCUSSION TITLE
Maintainer cannot change the publishing source for Custom GitHub Actions Workflows for Pages

### DISCUSSION THREAD

**Discussion by tnir** [community/community#27583] [Other Feature Feedback, Questions, & Ideas]
As per the beta feature (release: https://github.blog/changelog/2022-07-27-github-pages-custom-github-actions-workflows-beta/) of Custom GitHub Actions Workflows (beta) in GitHub Pages, people with maintainer permission cannot change the publishing source and see the current configuration if GitHub Actions is configured.

![](https://i0.wp.com/user-images.githubusercontent.com/14911070/178842638-51b834d3-6c54-423e-95fa-822f734fa98a.png?ssl=1)

Due to invisibility of it for maintainers, people with maintainer permission can revert the publishing source from GitHub Actions to branches.

### Suggestions
Even in beta, to maintain backward compatibility, you should allow people with maintainer permission to see the current status correctly and change it.

### References
https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site says:

> People with admin or maintainer permissions for a repository can configure a publishing source for a GitHub Pages site.

**Comment by yoannchaudet** [comment]
@tnir Thank you for the report. This is an omission (and a bug) that we are going to fix asap. Maintainers should be able to set the source to GitHub Actions just like a repo admin.

**Comment by yoannchaudet** [comment]
The fix should have landed in production by now. Let me know if you run into further issues.
